main.py

```diff
@@ -34,12 +43,12 @@ def _train(pipeline_name, dev_mode):
 
     logger.info('reading data in')
     if dev_mode:
```

Comment:
@pknut I think it would be better to have an option to choose whether russian or english model is trained. That could be done by adding a param in neptune use_english or use_english or something and then here one would just go if params.use_english: filepath_train =... filepath_test = ... filepath_valid =...